docker-compose.prod.yml

```diff
@@ -23,10 +29,10 @@ services:
     - db
     restart: always
     environment:
-      HASURA_GRAPHQL_DATABASE_URL: postgres://hello_zm:hello_zm@db:5432/hello_zm_dev
-      HASURA_GRAPHQL_ENABLE_CONSOLE: "true" # set to "false" to disable console
-      HASURA_GRAPHQL_ENABLED_LOG_TYPES: startup, http-log, webhook-log, websocket-log, query-log
-      HASURA_GRAPHQL_ADMIN_SECRET: dev_only_password
+      - HASURA_GRAPHQL_ADMIN_SECRET
```

Comment:
not needed if exist in `.env`

Reply:
Docker isn't automatically passing all env variables to all containers